app/ui/client/components/selectDropdown.js

```diff
@@ -1,23 +0,0 @@
-import { ReactiveVar } from 'meteor/reactive-var';
-import { Template } from 'meteor/templating';
-
-Template.selectDropdown.events({
-	'focus input'(e, i) {
-		i.open.set(true);
-		console.log('asdasd');
-	},
-	'blur input'(e, i) {
-		setTimeout(() => {
-			i.open.set(false);
-		}, 100);
-		console.log('asdasd');
-	},
-});
-Template.selectDropdown.helpers({
-	open() {
-		return Template.instance().open.get();
-	},
-});
-Template.selectDropdown.onCreated(function() {
-	this.open = new ReactiveVar(false);
-});
```

Comment:
Not needed. (Wow, this is obsolete!)

Reply:
The PR is removing this code, not adding it :P